tests/application_tests/test_env_var_changes.py

```diff
@@ -39,6 +41,9 @@ def setUp(self) -> None:
             self.client.delete_index(self.index_name_1)
         except MarqoApiError as s:
             pass
+
+    def tearDown(self) -> None:

```

Comment:
Could we have the marqo-os start/stop tests in a different class like `TestMarqoOSRetries`, so the marqo-os restart on `tearDown` only affects the needed tests?